build.gradle

```
@@ -420,7 +420,8 @@ launch4j {
     jreMinVersion = '11'
     jreMaxVersion = '21'
```

It works on Windows 11
```suggestion
    jreMinVersion = '11.0'
    jreMaxVersion = '21.0'
```